File: currency_conversion/src/endpoints/swagger.py
#--- Configure endpoint for swagger ---#

from flask_swagger_ui import get_swaggerui_blueprint

API_URL = '/api/swagger.json'
SWAGGER_URL = '/api/docs'

#setup blueprint and configs for swagger 
swagger_ui = get_swaggerui_blueprint(
    SWAGGER_URL,
    API_URL,
    config={
        'app_name': 'Currency Conversion App - Challenge Bravo'
    }
)

File: src/routes/swagger.js
module.exports = (app, accessProtectionMiddleware) => {

    const express = require('express');
    const router = express.Router();

    const swaggerUi = require('swagger-ui-express');
    const swaggerDocument = require('../../swagger.json');

    router.use('/', swaggerUi.serve);
    router.get('/', swaggerUi.setup(swaggerDocument));

    app.use('/api-docs', router);
};